src/polus/plugins/_plugins/classes/__init__.py

```diff
@@ -1,4 +1,17 @@
-from .plugin_classes import (ComputePlugin, Plugin, _Plugins, load_plugin,
-                             submit_plugin)
+"""Plugin classes and functions."""
+from polus.plugins._plugins.classes.plugin_classes import (ComputePlugin,
+                                                           Plugin, get_plugin,
+                                                           list_plugins,
+                                                           load_plugin,
+                                                           refresh,
+                                                           submit_plugin)
```

Comment:
This doesn't appear to be black formatted.

Reply:
Formatted with black and `isort --profile black`